labs/lab2/bigapp/Dockerfile.annotated

```diff
@@ -19,7 +19,7 @@ RUN yum -y update
 >>> cache in our intermediate cached image layer
 
 # Common Deps
-RUN yum -y install openssl procps-ng
```

Comment:
it probably should not have been removed. however if something else is already pulling in `procps-ng` then it's fine to remove it. can you `rpm -q --whatrequires procps-ng` ?